### PR TITLE
Update docs to use NPM package

### PR DESCRIPTION
**Pull request by asinner**
- use code snippets with example demonstrations to ensure reliability.
- pull file snippets (where applicable) from source instead of duplicating

### CODE REVIEW

**Comment by linweigao** [comment]
If this PR is merged, the local deployment will be an issue. Let's hold this PR until we have a clear solution about this issue.

**Comment by asinner** [comment]
Updated docs build to allow building components from source, see docs/README.md.

File: docs/package.json [hunk 1]
```diff
@@ -32,6 +33,7 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=prod webpack -p",
-    "start": "webpack-dev-server --history-api-fallback"
+    "start": "cross-env NODE_ENV=prod webpack-dev-server --history-api-fallback",
+    "start:development": "cross-env NODE_ENV=development webpack-dev-server --history-api-fallback"
```
Comment (linweigao) [inline docs/package.json]: Can we have "start" with local build?

Comment (linweigao) [inline docs/package.json]: "start:package"

**Review by linweigao — APPROVED** [review]

File: docs/package.json (hunk 1 above)
Comment (asinner) [reply docs/package.json]: done

Comment (asinner) [reply docs/package.json]: done

**Pull request merged**